scripts/zkai_attention_kv_fused_softmax_table_microprofile_gate.py

```
@@ -0,0 +1,920 @@
+    if row["fused_proof_size_bytes"] != row["proof_section_payload_bytes_total"] + row["proof_json_wrapper_bytes"]:
+        raise FusedSoftmaxTableMicroprofileGateError("proof byte total drift")
+    buckets = row["proof_byte_buckets"]
+    expected_bucket_keys = {
+        "commitment_bucket_bytes",
+        "query_bucket_bytes",
+        "opening_bucket_bytes",
+        "config_and_pow_bytes",
+        "json_wrapper_bytes",
+    }
+    if not isinstance(buckets, dict) or set(buckets) != expected_bucket_keys:
+        raise FusedSoftmaxTableMicroprofileGateError("proof bucket field drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)